web/src/components/Prices/_HollaexToken.scss

```diff
@@ -0,0 +1,254 @@
+.hollaex-token-wrapper {
+	height: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+
+	.wizard-container .ant-input-affix-wrapper,
+	.wizard-container .ant-input,
+	.ant-input-number,
+	.ant-select-single:not(.ant-select-customize-input) .ant-select-selector {
+		background-color: #202020;
+	}
+
+	.ant-select:not(.ant-select-disabled):hover .ant-select-selector,
+	.ant-select-focused:not(.ant-select-disabled).ant-select-single:not(.ant-select-customize-input)
+		.ant-select-selector {
+		border-color: #d9d9d9;
+	}
+
+	.subHeader-icon {
+		height: 20px;
+		width: 20px;
+		margin: 1px 3px;
+	}
+
+	.gray-text {
+		color: #b2b3b3;
+	}
+
+	.token-wrapper {
+		width: 65%;
+		height: 80%;
+
+		.image-container {
+			.coins-icon {
+				width: 3.5rem;
+				height: 3.5rem;
+			}
+		}
+
+		.link {
+			color: #5d63ff;
+			text-decoration: underline;
+			cursor: pointer;
+		}
+
+		.header-container {
+			width: 100%;
+			font-size: 16px;
+
+			.title {
+				font-size: 24px;
+				color: #b2b3b3;
+
+				span {
+					color: #ffffff;
+				}
+				.favourite-text {
+					font-size: 12px;
+					color: #707070;
+					padding-right: 10px;
+					opacity: 0;
+				}
+				.star-icon:hover {
+					.favourite-text {
+						opacity: 1;
+					}
+				}
+				.favourite-text-2 {
+					font-size: 12px;
+					color: #b2b3b3;
+					padding-right: 10px;
+				}
+			}
+		}
+
+		.ant-btn {
+			background: #242426;
+			color: #b2b3b3;
+		}
+
+		.hollaex-container {
+			border: 1px solid $colors-super-pale-black;
+
+			.token-container {
+				display: flex;
+				justify-content: space-between;
+
+				.info-container {
+					width: 65%;
+					background-color: #242426;
+					padding: 4rem;
+
+					.header-text {
+						font-size: 16px;
+						font-weight: bold;
+						letter-spacing: 1px;
+					}
+
+					.sub-text {
+						font-size: 14px;
+						width: 70%;
+						margin-top: 1rem;
+						color: #b2b3b3;
```

Comment:
Colors should be dynamic. Also, there are a couple of classes for the text color that I think can be used here.